services/identity-service/src/did/did.service.ts

```diff
@@ -1,13 +1,14 @@
 import { Injectable, InternalServerErrorException, Logger, NotFoundException } from '@nestjs/common';
-import { generateKeyPair } from '@decentralized-identity/ion-tools';
 import { PrismaService } from '../utils/prisma.service';
-import { DIDDocument } from 'did-resolver';
+const { DIDDocument } = require('did-resolver');
+type DIDDocument = typeof DIDDocument;
 import { uuid } from 'uuidv4';
 import { GenerateDidDTO } from './dtos/GenerateDid.dto';
 import { VaultService } from '../utils/vault.service';
 import { Identity } from '@prisma/client';
 @Injectable()
 export class DidService {
+  keys = {}
   static getKeySignType = (algo: string): any => {
     return {
       keyType: "JsonWebKey2020",
```

Comment:
Why do we still need this, can we not use the actual key type here?

Reply:
since digitalbazaar libraries required to be imported dynamically, this map is used to store them when we import on mount